provide-storage/backup-storage-provider-bsp.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Backup Storage Provider (BSP)
+description: Understand the responsibilities, operating flow, and reliability expectations for running a Backup Storage Provider on DataHaven.
+---
+
+# Backup Storage Provider (BSP)
+
+Backup Storage Providers supply redundancy for every bucket on DataHaven. They store replicated file bytes, maintain a verifiable record of the file keys they hold, answer periodic proof challenges, and serve data if a Main Storage Provider (MSP) becomes unavailable.
+
+## What a BSP Does
+
+- **Accept replication from MSPs**: Receive file chunks from MSPs after a user upload, validate integrity, and persist replicas until policy allows deletion.
+- **Maintain a global commitment**: Track stored file keys in a Merkle forest, publish the resulting root on-chain, and keep it current as files are added or removed.
+- **Answer protocol challenges**: Respond within the challenge window using Merkle proofs derived from local data to show continued custody. Missed or invalid answers can lead to slashing.
+- **Support recovery**: If an MSP fails, stream replicas to a replacement MSP so the bucket can be reconstructed without data loss.
+- **Manage capacity**: Advertise realistic storage limits and monitor utilization so you can continue accepting replication without breaching service targets.
+
+## Operational Flow
```

Comment:
Operational flow to do what? Provide BSP services? If I want the answer to this question, I have to read through each of the steps. ideally there's an intro sentence that tells users what happens if they follow these set of steps before diving into them